cmd/plan.go

```diff
@@ -67,7 +67,8 @@ func planHandler(cmd *cobra.Command, flags planFlags) {
 	planfile := flags.planfile
 	srcpath := flags.srcpath
 	name := flags.name
-
+	srcPathStruct := common.GetInputClonedPath(srcpath, common.RemoteSourcesFolder, true)
```

Comment:
Line 70 means the following
1. srcpath could be path on filesystem or git remote path
2. third argument is meant for overwrite and it is set to true
3. returned value srcPathStruct is Input struct that holds two values cloned filesystem path and git remote path.
4. this implementation is backward compatible. If the input is filesystem path from the user, then the input struct will hold the same filesystem path that user provided and remote git path will be an empty string.